tests/test_mantid.py

```diff
@@ -154,17 +154,17 @@ def test_Workspace2D_not_common_bins_masks(self):
             ds.masks["spectrum"].values[0:3],
             [True, True, True])
 
-    def test_Workspace2D_with_separate_monitors(self):
-        filename = MantidDataHelper.find_file("WISH00016748.raw")
-        ds = sc.neutron.load(filename,
-                               mantid_args={"LoadMonitors": "Separate"})
-        check_monitors(ds, 4471)
-
-    def test_Workspace2D_with_include_monitors(self):
-        filename = MantidDataHelper.find_file("WISH00016748.raw")
-        ds = sc.neutron.load(filename,
-                               mantid_args={"LoadMonitors": "Include"})
-        check_monitors(ds, 4471)
+#    def test_Workspace2D_with_separate_monitors(self):
```

Comment:
Can you use the `pytest` decorator to disable the test (explaining why, i.e., not enough memory?).